tests/test_utils.py

```
@@ -497,8 +497,8 @@ def quantity_score_func(x) -> Probability:
         [
             None,
             [],
-            [lambda x: True],  # Always satisfied constraint
-            [lambda x: x[0] >= 0.0],  # Trivial constraint
+            [lambda x: 1.0],  # Always satisfied constraint
```

Ruff (0.15.2)

[warning] Unused lambda argument: `x`

(ARG005)